lib/registries/addon/overview/index/controller.ts

```diff
@@ -11,13 +10,4 @@ export default class Overview extends Controller {
 
     @alias('model.taskInstance.value')
     registration?: Registration;
-
-    @computed('registration.{root,isRoot}')
-    get root() {
-        if (!this.registration || this.registration.isRoot) {
-            return undefined;
-        }
-
-        return this.registration.belongsTo('root').value() as Registration;
```

Comment:
The purpose of using `this.registration.belongsTo('root').value()` is because `root` is an asynchronous relationship . For an asynchronous relationship, using `this.registration.root` directly will trigger another AJAX call to fetch the `root` relationship, which in this case shouldn't be necessary because the `root` relationship is already embedded in the response to fetch the `registration` itself.

Reply:
@adlius Looking at the code, I had removed this function completely so it did not make the additional call. It is embedded in the registration, so I called it directly on the template. I had explained that previously in scrum, did you have another optimization to suggest or would like to see it fixed differently than what was already completed?

Reply:
Removing this `computed` property would not stop it from an additional AJAX call, since the code itself should not be making any AJAX call. The actual trigger of the additional AJAX call to fetch the `root` relationship is in the template.

Reply:
Ok, I just commented below which also addresses this.